fdk_rdf_parser_service/config.py

```diff
@@ -69,3 +70,13 @@ class BlackboxExporterFilter(logging.Filter):
     def filter(self: Any, record: logging.LogRecord) -> bool:
         """Filter function."""
         return "Blackbox Exporter" not in record.getMessage()
+
+
+class RdfLibFailedXsdDateConversionFilter(logging.Filter):
```

Comment:
Selv om det er date som i hovedsak skjer så vil denne fikse mer enn bare det
```suggestion
class RdfLibFailedXsdConversionFilter(logging.Filter):
```